### ISSUE TITLE
Don't link to english only resources in translated strings

### ISSUE THREAD

**Issue by bhousel** [openstreetmap/iD#2940]
per [this issue on transifex](https://www.transifex.com/ideditor/id-editor/translate/#pt/core/15431540)

We should separate out urls from translation text and give each url its own key so that translators can specify an appropriate target url.

- [x] `success.help_html`

@kepta this relates to #2938, already discussed w you..

**Comment by kepta** [comment]
Does that mean what core.yaml is to text
we want a file which has all the link?

**Comment by bhousel** [comment]
> Does that mean we create a new yaml file which handles links separately ?

I think we should split `success.help_html` into: 

```
    help_text: Your changes should appear in the "Standard" layer in a few minutes. Other layers, and certain features, may take longer. ({link})
    help_link_text: details
    help_link_url: https://wiki.openstreetmap.org/wiki/FAQ#I_have_just_made_some_changes_to_the_map._How_do_I_get_to_see_my_changes.3F
```

This 
1. avoids embedding html links `<a>` in translated text, which is a security risk
2. lets translators change just the url without changing the text
3. encourages translators to translate the wiki page

**Comment by kepta** [comment]
>> avoids embedding html links <a> in translated text, which is a security risk 
:+1:

**Comment by bhousel** [comment]
merged as 427d08a
